tutorials/intellijJUnit.md

```diff
@@ -0,0 +1,95 @@
+{% set title="IntelliJ - Using the JUnit" %}
+<frontmatter>
+  title: "{{ title }}"
+  pageNav: 2
+</frontmatter>
+
+<include src="../common/common-fragments.md#wip-warning" />
+
+# {{ title }}
+
+This tutorial covers the basics of setting up and using JUnit with IntelliJ.
+
+<!-- ======================================================== -->
+
+## Prerequisites
+
+<include src="vscCreatingNewJavaProject.md#vsc-java-prereq" />
+
+<include src="junit.md#junit-use-gradle" />
+
+This tutorial assumes you are using Gradle to manage JUnit dependencies.
+
+<!-- ======================================================== -->
+
+## Setting Up JUnit with Gradle
+
+#### Add Gradle Support
+
+If your project doesn’t yet use Gradle, follow the [Gradle Tutorial](https://se-education.org/guides/tutorials/gradle.html) to set it up.
+
+#### Add JUnit as a Dependency
+
+1. Open the `build.gradle` file in your project’s root directory.
+
+<box type="tip" seamless>
+
+Press <kbd>⌘Cmd</kbd> <kbd>⇧Shift</kbd> <kbd>O</kbd> and search for `build.gradle` to locate it quickly.
```

Comment:
If we optimise for one OS, that needs to be Windows, not Mac. Ideally, should give info for both.